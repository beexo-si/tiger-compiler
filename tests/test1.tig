/* probando assign y trdec caso VarDec */

let var a := 5
	var b := 3
in b
end
