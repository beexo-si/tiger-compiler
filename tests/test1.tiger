/* probando assign */

let var a := 5
	var b := 3
in b
end
