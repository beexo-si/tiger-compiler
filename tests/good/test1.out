bien!
yes!!
Return code: 3
